ai_traineree/agents/dummy.py

```diff
@@ -0,0 +1,178 @@
+import copy
+import numpy as np
+from ai_traineree.agents import AgentBase
+from ai_traineree.types import AgentState, BufferState, DataSpace, NetworkState
+from typing import Callable, Dict, Optional, Type
+from ai_traineree.loggers import DataLogger
+from ai_traineree.experience import Experience
+
+import warnings
+
+class DummyAgent(AgentBase):
+    """Deep Q-Learning Network (DQN).
+     
+     Agent that returns random values regardless of the input
+     Print mean of the taken actions, observables and reward
+     
+     """
+    model = "Dummy"
+    def __init__(        self,
+        obs_space: DataSpace,
+        action_space: DataSpace,
+        state_transform: Optional[Callable] = None,
+        reward_transform: Optional[Callable] = None,
+        **kwargs
+    ):
+        """Initiates Dummy agent.
+
+        Parameters:
+            obs_space (DataSpace): Dataspace describing the input.
+            action_space (DataSpace): Dataspace describing the output.
+            state_transform (optional func): Function to transform (encode) state before used by the network.
+            reward_transform (optional func): Function to transform reward before use.
+
+        """
+        super().__init__(**kwargs)
+        self.obs_space = obs_space
+        self.action_space = action_space
+        self.iteration: int = 0
+
+        self._loss: float = float("nan")
+
+    @property
+    def loss(self):
+        return {"loss": self._loss}
+
+
+    def reset(self):
+        warnings.warn("Dummy Agent is for debugging purposes only. Nothing to reset.")
+        return
+
+
+    def step(self, exp: None):
```

Comment:
I'm not sure for the reason but it seems that you're adding `None` type in all  methods' signature for this class. Although that isn't syntax error it doesn't bring any value. I'd suggest to either skip adding type or add the correct one.

Reply:
I see. My mistake, I updated the file. Thanks for letting me know :)